docs/modules/devicetwin.md

```diff
@@ -0,0 +1,328 @@
+
+# DeviceTwin
+
+
+## Overview
+
+DeviceTwin is responsible for storing device status and syncing device status to the cloud. 
```

Comment:
dealing with device status in one of the jobs of device twin. reading later through the document, we can see it also deals with device twin attributes, membership events etc. Can we describe all of the responsibilities of device twin in a summary?